classes/helper.php

```diff
@@ -98,11 +98,13 @@ public static function enable_notifications() : string {
         if (get_capability_info('local/csp:seenotifications')) {
             $cansee = has_capability('local/csp:seenotifications', $PAGE->context, $USER->id);
         }
-        if (!$cansee | ($conf->notifications_enable_enforced + $conf->notifications_enable_reported == 0)) {
+        $notificationsenableenforced = $conf->notifications_enable_enforced ?? 0;
+        $notificationsenablereported = $conf->notifications_enable_reported ?? 0;
+        if (!$cansee || ($notificationsenableenforced + $notificationsenablereported == 0)) {
             return '';
         }
-        $collect_enforced = $conf->notifications_enable_enforced == 1 ? 'true' : 'false';
-        $collect_reported = $conf->notifications_enable_reported == 1 ? 'true' : 'false';
+        $collectenforced = $notificationsenableenforced == 1 ? 'true' : 'false';
```

Comment:
are these supposed to be booleans, or strings containing a word 'true' ? I don't even thing we need the ternary

Reply:
I just left it how it was, but yeh, looks silly. I removed those variables completely.